src/app/layout.tsx

```diff
@@ -9,26 +9,27 @@ import localFont from "next/font/local";
 const myFont = localFont({
   src: [
     {
-      path: "./fonts/BentonSansBook.otf",
-      weight: "300",
+      path: "./../app/fonts/BentonSansBook.otf",
```

Comment:
Can we move the font files to the `public/` folder (or an `assets/` folder) in order to keep the Next.js file routing directory tree "clean"? You should be able to import the fonts in the same way using relative file paths.